keyboards/adpenrose/mine/config.h

```diff
@@ -0,0 +1,15 @@
+// Copyright 2022 Arturo Avila (@Arturo Avila)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+/* Solenoid pin */
+#define SOLENOID_PIN A7
+
+#define DYNAMIC_KEYMAP_LAYER_COUNT 2
```

Comment:
```suggestion
#ifndef VIA_ENABLE
#    define DYNAMIC_KEYMAP_LAYER_COUNT 2
#endif
```

Reply:
That was a mistake of mine when coding. Due to the available EEPROM on the chip that I'm using, the keyboard can only work if the maximum number of layers is two. I've updated the VIA keymap so that it only has two layers.